frontend/src/components/story/StoryCard.css

```diff
@@ -0,0 +1,44 @@
+.story-card {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+
+    width: 80%;
```

Comment:
Should min width be % or vw or px? Should it be on the `.story-card` class?

Reply:
let's do px since it's the same across devices. 
`.story-card` is a good home since we'd be specifying the min-width of the entire card to prevent the vertically stacked issue.